src/mono/wasm/runtime/exports.ts

```diff
@@ -117,17 +121,20 @@ function export_to_emscripten(dotnet: any, mono: any, binding: any, internal: an
     Object.assign(binding, BINDING);
     Object.assign(internal, INTERNAL);
 
-    // backward compatibility, sync with EmscriptenModuleMono
-    Object.assign(module, {
-        // https://github.com/search?q=mono_bind_static_method&type=Code
-        mono_bind_static_method: (fqn: string, signature: ArgsMarshalString): Function => {
+    // this could be overriden on Module
+    moduleExt.preInit = mono_wasm_pre_init;
+    moduleExt.onRuntimeInitialized = mono_wasm_on_runtime_initialized;
+
+    if (!moduleExt.disable6Compatibility) {
```

Comment:
did we invent this name? if so, can we prefix it somehow?

Reply:
Let's go with `disableDotNet6Compatibility`